DevEdu.API/DevEdu.Business/Configuration/AuthOptions.cs

```diff
@@ -0,0 +1,17 @@
+﻿using System.Text;
+using Microsoft.IdentityModel.Tokens;
+
+namespace DevEdu.Business.Configuration
+{
+    public class AuthOptions
```

Comment:
we have the same in API-project
so please remove this class from BLL

Reply:
I use it in BLL, that has no dependency from API, should I delete it from API?

Reply:
replaced to BLL